pkg/storaged/alerts.jsx

```diff
@@ -0,0 +1,68 @@
+/*
+ * This file is part of Cockpit.
+ *
+ * Copyright (C) 2024 Red Hat, Inc.
+ *
+ * Cockpit is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation; either version 2.1 of the License, or
+ * (at your option) any later version.
+ *
+ * Cockpit is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import React from "react";
+import { AlertGroup, Alert, AlertActionCloseButton } from "@patternfly/react-core/dist/esm/components/Alert/index.js";
+
+import { useEvent } from "hooks.js";
+
+class GlobalAlerts extends EventTarget {
+    constructor() {
+        super();
+        this.alerts = [];
+    }
+
+    emit_changed() {
+        this.dispatchEvent(new CustomEvent("changed"));
```

Comment:
These 2 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-20061-20240219-162228-2dbcaaa3-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)